distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java

```diff
@@ -223,6 +223,18 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
         return options(junitBundles(),
                 // HACK: incorrect version exported to override hamcrest-core from exam
                 // feature which causes a split package issue for rest-assured
+                wrappedBundle(mavenBundle("org.apache.httpcomponents",
+                        "com.springsource.org.apache.httpcomponents.httpclient")
```

Comment:
Maybe my earlier comment in the pom file was misunderstanding on my part. Are the `com.springsource.org.apache.httpcomponents.httpclient` and `httpclient` artifacts not the same but with different names on the newer versions? (Same question re: the two `httpcore` artifacts.)